README.md

```diff
@@ -245,6 +256,38 @@ Used for estimating sizes of macros with long build times and checking if they w
 
 These targets print RAM summaries for a given module.
 
+#### GUI Targets
+
+Those targets are used to run OpenROAD GUI with Docker flow. They are executable rules, therefore have to be called with `run`:
+
+```
+bazel run L1MetadataArray_full_final_gui
```

Comment:
This name is too complicated. What does "full" mean here?

Reply:
This was added as another variant for `L1MetadataArray` without mocking to test stages like `route` or `final`, which are not available for `L1MetadataArray_test`

Reply:
Aha!